modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -595,3 +609,17 @@ conf = {{{packageName}}}.Configuration(
             url = url.replace("{" + variable_name + "}", used_value)
 
         return url
+
+    @property
+    def host(self):
+        """Return generated host."""
+        if self.server_index is None:
+            return self._base_path
```

Comment:
How about removing this if block because it already exists in get_host_from_settings?